packages/firebase_snippets_app/lib/snippets/firestore.dart

```diff
@@ -102,9 +104,26 @@ class FirestoreSnippets extends DocSnippet {
     // [END data_model_sub_collections]
   }
 
-  void dataBundles_loadingClientBundles() {
+  void dataBundles_loadingClientBundles() async {
     // [START data_bundles_loading_client_bundles]
-    // TODO - currently in scratch file
+    // Get a bundle from a server
+    final url = Uri.https('example.com', '/create-bundle');
+    final response = await http.get(url);
+    String string = response.body;
+    final buffer = Uint8List.fromList(string.codeUnits);
+
+    // Load a bundle from a buffer
+    LoadBundleTask task = FirebaseFirestore.instance.loadBundle(buffer);
+    await task.stream.toList();
+
+    // Use the cached named query
+    final results = await FirebaseFirestore.instance.namedQueryGet(
+      "example-query",
```

Comment:
The Web, iOS and Android samples use `latest-stories-query` (see [example](https://github.com/firebase/snippets-android/blob/e69c30ac8e108c7f1e9912c8d5f764911d38057e/firestore/app/src/main/java/com/google/example/firestore/kotlin/SolutionBundles.kt#L42)) - let's use that for consistency across languages.

```suggestion
      "latest-stories-query",
```